sources/Application/Player/Player.cpp

```diff
@@ -529,32 +540,33 @@ void Player::Update(Observable &o, I_ObservableData *d) {
       }
     }
 
-    // Do we need to kill a voice ?
-
-    if (sync->TableSlice()) {
-      for (int i = 0; i < SONG_CHANNEL_COUNT; i++) {
-        bool stopped = false;
-        if (timeToLive_[i] > 0) {
-          if (--timeToLive_[i] == 0) {
-            mixer_->StopInstrument(i);
-            stopped = true;
+    // Do we need to kill a voice?
+    if (viewData_->playMode_ != PM_AUDITION) {
```

Comment:
is this needed because the voice stealing code interferes with auditioning?

Reply:
Well, tbh I'm not sure what this does, but it seemed not to be instrument specific so I left it out. Do you have insight on it?

Reply:
All good then. I dont really know about it either apart from that it seems to be voice stealing code: that is when max number of voices is reached, it chooses which existing voice to stop to allow new voice to play required note.

Reply:
I've removed this and haven't found perceptible difference, so...